### PR TITLE
New blog post: Review and Setup Guide for UNIT3D

### LINKED ISSUE #235: New blog post: Review and Setup Guide for UNIT3D
I've installed [UNIT3D](https://github.com/HDInnovations/UNIT3D-Community-Edition). It's a BitTorrent Index with many features.

I will write some recommendations for the installation and review some of its features. I won't be a detailed review. I'm not an expert in BitTorrent Indexes, and I don't know most of its features. In fact, my first goal is to learn more about the indexes that are being used nowadays.

### PR DESCRIPTION
**Pull request by josecelano**
New blog post: Review and Setup Guide for UNIT3D.

### CODE REVIEW

File: project-words.txt [hunk 1]
```diff
@@ -82,9 +86,12 @@ signup
 Stackoverflow
 striptags
 struct
+themoviedb
 tlsv
+TMDB
 torrenting
 Torrust
+Troubleshoting
```
Comment (Roardom) [inline project-words.txt:94]: Should be "Troubleshooting".

File: src/routes/(blog-article)/review-and-setup-guide-for-unit3d/+page.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+
+- Server (virtual machine)
+- External Services
+- System requirements (software installed on the server)
+
+#### Hardware Requirements (server)
+
+We opted for using a Digital Ocean droplet just because we are used to their interface and it's straightforward. Feel free to use your favorite provider.
+
+In our first attempt we used the minimum size image. However it was to small. The server had problems building the frontend and that's one of the last steps in the installation. Finally we used a VM (Digital Ocean call them droplets) with these characteristics:
```
Comment (Roardom) [inline src/routes/(blog-article)/review-and-setup-guide-for-unit3d/+page.md]: "to small" -> "too small"

File: src/routes/(blog-article)/review-and-setup-guide-for-unit3d/+page.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+
+Thats it. If all went fine in during the installation you can load the login page:
+
+<Image src="/images/posts/review-and-setup-guide-for-unit3d/unit3d-login-page.png" alt="UNIT3D Login Page" />
+
+You can use the owner account credentials you provided during the installation.
+
+<Callout type="info">
+
+If the official demo is down you can apply for signing up in our demo on: <https://www.unit3d-demo.com/application>. You don't need to provide real data, just a valid mail so we can confirm it manually. We could sut down this demo at any time.
```
Comment (Roardom) [inline src/routes/(blog-article)/review-and-setup-guide-for-unit3d/+page.md]: "sut" -> "shut"

**Review by Roardom — COMMENTED** [review]
Looks good! I just noticed a few typos (there might be a few more I missed)

**Comment by josecelano** [comment]
ACK 65ac9f8467ceb0ac554ac18514bcaf5d44dcfdd6

**Pull request merged**